src/kwio.sh

```diff
@@ -28,9 +28,19 @@ function alert_completion()
 
   grep -o . <<< "$opts" | while read option;  do
     if [ "$option" == "v" ]; then
-      eval "${configurations[visual_alert_command]} &"
+      if command_exists "${configurations[visual_alert_command]} &"; then
+        eval "${configurations[visual_alert_command]} &"
+      else
+        complain "The command set in the visual_alert_command variable" \
+        "couldn't be run. Check if the necessary packages are installed."
```

Comment:
Use `warning` instead of `complain`.

I think it would be nice to show the value inside `${configurations[visual_alert_command]}` for the user. Same for the sound alert.